lib/mayaUsd/commands/Readme.md

```diff
@@ -153,6 +154,7 @@ their own purposes, similar to the Alembic export chaser example.
 | `-materialsScopeName`            | `-msn`     | string           | `Looks`             | Materials Scope Name |
 | `-mergeTransformAndShape`        | `-mt`      | bool             | true                | Combine Maya transform and shape into a single USD prim that has transform and geometry, for all "geometric primitives" (gprims). This results in smaller and faster scenes. Gprims will be "unpacked" back into transform and shape nodes when imported into Maya from USD. |
 | `-normalizeNurbs`                | `-nnu`     | bool             | false               | When setm the UV coordinates of nurbs are normalized to be between zero and one. |
+| `-preserveUVSetNames`            | `-puv`     | bool             | false               | Refrain from renaming UV sets additional to "map1" to "st1", "st2", etc; "map1" will always be renamed to "st". |
```

Comment:
I think this comment may be improved:
1. In the code, -remapUVSetsTo takes priority over -preserveUVSetNames. We need to add a comment here clarifying this behaviour.
2. The comment here says: "map1" will always be renamed to "st". I'm not sure I saw this behaviour in the code. Is this true?

Reply:
Yes sorry that's a bit outdated. I'll go ahead and fix that right up.